app/models/legislation/draft_version.rb

```diff
@@ -22,9 +22,16 @@ class Legislation::DraftVersion < ApplicationRecord
   scope :published, -> { where(status: "published").order("id DESC") }
 
   def body_html
-    renderer = Redcarpet::Render::HTML.new(with_toc_data: true)
-
-    Redcarpet::Markdown.new(renderer).render(body)
+    # See https://github.com/vmg/redcarpet for options
```

Comment:
At Consul we try to avoid putting comments in the code, the commit message might be a good place to add this reference to the options offered by RedCarpet 🤔